include/RAJA/policy/sequential/forall.hpp

```diff
@@ -88,16 +88,29 @@ namespace impl
 //
 
 template <typename Iterable, typename Func>
-RAJA_INLINE void forall(const seq_exec &, Iterable &&iter, Func &&loop_body)
+#if defined(RAJA_COMPILER_GNU)
+RAJA_GNU_NoSIMD
+#else
+RAJA_INLINE
+#endif
+void forall(const seq_exec &, Iterable &&iter, Func &&loop_body)
 {
   auto end = std::end(iter);
+
+  RAJA_NoSIMD
   for (auto ii = std::begin(iter); ii < end; ++ii) {
     loop_body(*ii);
   }
 }
 
+
 template <typename Iterable, typename Func, typename IndexType>
-RAJA_INLINE typename std::enable_if<std::is_integral<IndexType>::value>::type
+#if defined(RAJA_COMPILER_GNU)
```

Comment:
This seems confusing. One branch of the conditional relates to simd, the other relates to inline.